msteams-platform/apps-in-teams-meetings/build-tabs-for-meeting.md

```diff
@@ -176,6 +176,133 @@ TeamsJS is a rich library used to create Tabs using JavaScript. Use the latest T
+
+> [!NOTE]
+> If the app caching isn't enabled, the webview is recreated every time the users go out and return to the app.
+
+Following are the parameters to control the conditions for the apps to be added or removed from the cache (based on the configuration updates, the parameters can be modified):
+
+* If the system memory load is high, the app is removed from the cache.
+* If the app exceeds maximum cache size, the app is removed from the cache.
+* If the user doesn't return to the app within 20 minutes, the app is removed from the cache.
+* The app is not cached if Teams doesn't receive the `readyToUnload` signal from the app within 30 seconds after sending the `beforeUnload` notification.
+* App caching is disabled if the system memory is less than 4 GB.
+* Side panel is the only supported FrameContext for app caching in meetings.
+* App caching isn't supported for meetings where the invited user count is more than 20.
+
+### Code example
+
+The following code snippet is an example for `readyToUnload` callback function:
+
+```javascript
+// Using readyToUnload callback function
+const beforeUnloadHandler = (readyToUnload: () => void) => {
+  readyToUnload();
+  return true;
+};
+```
+
+The following code snippet is an example of `teamsCore.registerBeforeUnloadHandler` and `teamsCore.registerOnLoadHandler` APIs:
+
+```javascript
+microsoftTeams.app.initialize().then(() => {
+
+    microsoftTeams.teamsCore.registerBeforeUnloadHandler((readyToUnload) => {
+        const result = beforeUnloadHandler(readyToUnload);
+        // dispose resources and then invoke readyToUnload
+        console.log("got beforeunload from TEAMS");
+        return result;
+    });
+
+    microsoftTeams.teamsCore.registerOnLoadHandler((data) => {
+        console.log("got load from TEAMS", data.contentUrl, data.entityId);
+        // use contentUrl to route to correct page 
+        // invoke notifySuccess when ready  
+        app.notifySuccess();
+    });
+});
+```
+
+### Limitations
+
+The following are the limitations for app caching:
+
+* Single-page apps that use client-side routing for page navigation can benefit from app caching. It's recommended that the same domain is used across all contexts of your app launch.
+
+* Apps need to re-register for events such as `themeChange`, `focusEnter`, and so on, in the load handler. Teams client won't send any notifications to the app when cached. If your app requires notifications even when cached, caching might not be the right solution.
+
+* App caching is supported only in Teams desktop client. In Teams web client, even if the app registers load handlers, the app is removed from the cache after the unload sequence is completed.
+
+* Register the `load` and `beforeUnload` handlers early in your launch sequence. If the Teams client doesn’t identify these registrations before the user goes out of the app, the app isn't cached.
```

Comment:
change "goes out of " to leaves or navigates away

Reply:
As per Microsoft standard, we won't use words "Leaves or Navigate", so we used "go out of the app".